jib-maven-plugin/src/test/resources/maven/projects/simple/pom-jar-containerization.xml

```diff
@@ -37,7 +37,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-jar-plugin</artifactId>
-        <version>3.1.1</version>
+        <version>3.2.0</version>
```

Comment:
`maven-jar-plugin`'s latest version is 3.2.0. I'm assuming we're intentionally updating it here?

Reply:
Yeah, `maven-jar-plugin` does have 3.1.1, but I think we can always use the latest.